expression/schema.go

```diff
@@ -208,15 +207,6 @@ func (s *Schema) ColumnsByIndices(offsets []int) []*Column {
 	return cols
 }
 
-// GetTypes creates a field type slice.
-func (s *Schema) GetTypes() []*types.FieldType {
```

Comment:
Why remove this? There are some places in your code that can be replaced by this function.

Reply:
PTAL @lamxTyler